Server/htdocs/AppController/commands_RSM/utilities/RSMverifyBody.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+//***************************************************************************************
+// Structure
+//***************************************************************************************
+
+function checkBodyIsJsonObject($body)
+{
+  global $RSallowDebug;
+  if (!is_object($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body must be a JSON object '{}'");
+    else returnJsonMessage(400, "");
+  }
+}
```

Comment:
The name of this function is misleading. This function does not check a "body" but something that is being passed as parameter. The function should be named "checkIsJsonObject"